installing_on_kubernetes/_topics/pods_ssl.md

```diff
@@ -0,0 +1,74 @@
+## Using TLS to encrypt connections between pods inside the cluster:
+
+### Create a secret containing all of the certificates
+The certificates should all be signed by a CA and that CA certificate should be in uploaded as `root_crt` so that it can be used to verify connection validity.  If the secret is named `internal-certificates-secret`, no changes are needed in the CR, if you choose a different name, that should be set in the `internalCertificatesSecret` field of the manageiq CR.
+
+        oc create secret generic internal-certificates-secret \
+          --from-file=root_crt=./root.crt \
+          --from-file=httpd_crt=./httpd.crt \
+          --from-file=httpd_key=./httpd.key \
+          --from-file=memcached_crt=./memcached.crt \
+          --from-file=memcached_key=./memcached.key \
+          --from-file=postgresql_crt=./postgresql.crt \
+          --from-file=postgresql_key=./postgresql.key \
+          --from-file=ui_crt=./ui.crt \
+          --from-file=ui_key=./ui.key
+
+### If you need a script to generate certificates:
+The following is an example in ruby:
+
+        require 'openssl'
```

Comment:
If we wrap this in triple-backtick Ruby, it should be rendered with hightlights.  Alternatively, I think it would be better to put this code into the manageiq-pods repo in a `samples` or `tools` directory or something and then just link it here.

Reply:
Done